README.md

```diff
@@ -249,6 +260,15 @@ console.log(renderTime.current.Component.updates); // output: []
 **Note**: If you have some same component, these components combine to `array`  
 **Note**: Each time are displayed with `ms`
 
+### wait
+
+`wait` method is feature to wait for `renderCount` or `renderTime` is set. You need to wrap all `Matcher` is using `renderCount` or `renderTime` because we are assign `renderCount` and `renderTime` in asynchronous.  
```

Comment:
```suggestion
`wait` method is feature to wait for `renderCount` or `renderTime` is assigned. You need to wrap all returned value from `perf()` because we are assigning `renderCount` and `renderTime` in asynchronous.  
```